### PR TITLE
Fix google places autocomplete using enter key to select

### PR DESCRIPTION
**Pull request by clairesunstudio**
This PR is for updating Mayflower artifacts and adding a changelog after https://github.com/massgov/mayflower/pull/1580 gets merged into develop

To test, a locally working google API key must be used in openmass:

- $auth = ['key' => $this->key]; --> $auth = ['key' => '[google_api_key']; (see JIRA ticket for local API key)
- Build patternlab locally rush build:patternlab
- Clear cache in openmass ddev start && ddev drush cr
- Go to a location listing page https://www.mass.gov/orgs/massachusetts-registry-of-motor-vehicles/locations?icons=All&helper=Boston%2C+MA%2C+USA&lat=&lng=
- Search for a location in MA, and select an option from the drop down and press ENTER
- Verify that the list and the markers update accordingly

Refer to https://github.com/massgov/mayflower/pull/1580 for more testing instructions

----

To test on feature5:
- [ ] Go to a location listing page:
  - Before: https://www.mass.gov/orgs/massachusetts-registry-of-motor-vehicles/locations?icons=All&helper=Boston%2C+MA%2C+USA&lat=&lng=
  - After: https://massgovfeature5.prod.acquia-sites.com/orgs/massachusetts-registry-of-motor-vehicles/locations?icons=All&helper=&lat=&lng=
- [ ] Search for a location in MA, and select an option from the drop down and press ENTER
- [ ] Verify that the list and the map markers update accordingly

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
 # Ignore Drupal files
-docroot/sites/*/settings.local.php
+docroot/sites/**/settings.local.php
 docroot/sites/*/files
 docroot/sites/simpletest
-docroot/sites/*/secrets.settings.php
+docroot/sites/**/secrets.settings.php
 drush/sites/self.site.yml
-docroot/sites/*/services.local.yml
+docroot/sites/**/services.local.yml
```
Comment (Javi-er) [inline .gitignore]: @clairesunstudio do these need to be merged too? if not, please revert it.

File: composer.json [hunk 2]
```diff
@@ -241,7 +241,7 @@
         "friends-of-behat/mink-debug-extension": "^2",
         "furf/jquery-ui-touch-punch": "dev-master",
         "geocoder-php/open-cage-provider": "^4.4",
-        "massgov/mayflower-artifacts": "dev-develop",
+        "massgov/mayflower-artifacts": "dev-patternlab/DP-23780-fix-location-filter-key-enter",
```
Comment (Javi-er) [inline composer.json]: @clairesunstudio I think this change and the changes to composer.lock can be reverted as well.

File: .gitignore (hunk 1 above)
Comment (clairesunstudio) [reply .gitignore]: yes, this is fixing the gitignore, without this, it's not ignoring the file as expected.

File: composer.json (hunk 2 above)
Comment (clairesunstudio) [reply composer.json]: Yep, I will merge in the MF PR and update the composer.lock

**Pull request merged**